src/Components/Register/Register.tsx

```diff
@@ -1,7 +1,246 @@
-import React from 'react'
+import { useTheme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
 
-export default function Register() {
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+
+import Typography from "@mui/material/Typography";
+
+
+
+import Button from "@mui/material/Button";
+import Photo from "../../assets/Group (1).png";
+import "../../App.scss";
+
+import TextField from "@mui/material/TextField";
+import { Link, useNavigate } from "react-router-dom";
+import { FormControl, Stack } from "@mui/material";
+import { useContext, useState } from "react";
+import { AuthContext } from "../Context/AuthContext/AuthContext";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { toast } from "react-toastify";
+import CircularProgress from "@mui/material/CircularProgress";
+import { IRest } from "./ResetInterface/ResetInterface";
+
+
+export default function Register({ saveAdminData }) {
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<IRest>();
+  const [isLoading, setIsLoding] = useState(false);
+  const {baseUrl} = useContext(AuthContext);
+  const theme = useTheme();
+  // ****************** to register-Password **********************
+  const onSubmit = (data: IRest) => {
+    setIsLoding(true);
+
+    axios
+      .post(`${baseUrl}/admin/users`, data)
+
+      .then((response) => {
+        // console.log(response);
+        navigate("/login");
+        toast.success("Successfully");
+      })
+      .catch((error) => {
+        console.log(error);
+        toast.error(error);
+        setIsLoding(false);
+      });
+  };
   return (
-    <div>Register</div>
-  )
+    <>
+      <Stack spacing={30} direction="row" justifyContent="center">
+        <Box sx={{ display: "flex", flexDirection: "column" }}>
+          <CardContent sx={{ flex: "1 0 auto", marginLeft: 5 }}>
+            <Typography
+              sx={{ mb: 1 }}
+              component="div"
+              variant="h5"
+              className="text"
+            >
+              Stay<span className="text2">cation.</span>
+            </Typography>
+            <Box sx={{ marginLeft: 5, }}>
+              <Typography variant="h3" component="div" sx={{ mb: 2 }}>
+              Sign up
+              </Typography>
+              <Typography variant="h6" component="div" className="text3">
+                If you don’t have an account register <br />
+                You can <span>Login here !</span>
+              </Typography>
+              <FormControl
```

Comment:
The registration form is not complete, we need to add the missing required fields.